randovania/games/prime1/json_data/Chozo Ruins.txt

```diff
@@ -1525,13 +1525,21 @@ Extra - aabb: [41.40501, 247.92514, -8.917927, 128.55772, 335.16376, 23.842714]
       Trivial
   > Door to Transport Access North
       Trivial
+  > Event - Fallen Rubble
```

Comment:
Since door to transport access north can reach missile launcher without passing through your new event, logic is bugged.

Reply:
but the event is triggered upon getting the pickup? How is that bugged if it's the pickup that passes to the event and back?

Reply:
What about it should be changed? I've currently now made it impossible to go from pickup to Totem Access, and added a route from Transport Access North to Totem Access for after the event